packages/vips/package.json

```diff
@@ -31,5 +31,6 @@
 	},
 	"publishConfig": {
 		"access": "public"
-	}
+	},
+	"wpScript": true
```

Comment:
This was not in the original PR, this package was added in the interim in #64845.